Makefile

```diff
@@ -1,10 +1,17 @@
-all: modulo_lib.o modulo_lib_test.o
+all: modulo_lib_test
 
 modulo_lib.o: modulo_lib.cc modulo_lib.h 
-	g++ -c $< -o $@
+	g++ -c modulo_lib.cc -o $@
 
 modulo_lib_test.o: modulo_lib_test.cc modulo_lib.h 
-	g++ -c $< -o $@
+	g++ -c modulo_lib_test.cc -o $@
+
+modulo_lib_test: modulo_lib.o modulo_lib_test.o
+	g++ modulo_lib.o modulo_lib_test.o -o $@
```

Comment:
Here, you can use the automatic make variable $^ to refer to all prerequisites :)

Reply:
Cool! Done.

Reply:
Cool! Done.